### PR TITLE
Implemented Commenting

### PR DESCRIPTION
**Pull request by KlausBenislavskis**

### CODE REVIEW

File: Api/Functions/Comments.cs [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        }
+        [FunctionName("GetComments")]
+        [OpenApiOperation(operationId: "GetComments", tags: new[] { "Comments" })]
+        [OpenApiParameter(name: "movieId", In = ParameterLocation.Path, Required = true, Type = typeof(int))]
+        [OpenApiParameter(name: "page", In = ParameterLocation.Path, Required = true, Type = typeof(int))]
+        [OpenApiResponseWithBody(statusCode: HttpStatusCode.OK, contentType: "application/json", bodyType: typeof(List<Core.Models.Comment>))]
+        [OpenApiParameter(name: "x-functions-key", In = ParameterLocation.Header, Required = true, Type = typeof(string), Description = "The function key")]
+
+        public async Task<IActionResult> GetComments(
+            [HttpTrigger(AuthorizationLevel.Function, "get", Route = "comments/{movieId}/{page}")] HttpRequest req,
```
Comment (lukisko) [inline Api/Functions/Comments.cs:37]: I think we can discuss if the page parameter will be in route this way or as query so it will be consistent

Comment (KlausBenislavskis) [reply Api/Functions/Comments.cs:37]: sure

File: Infrastructure/Repositories/CommentRepository.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        {
+            using var db = GetQuantityDbUserConnection();
+            await db.InsertAsync(DomToDb.Map(comment, movieId, userId));
+        }
+
+        public async Task<List<Comment>> GetComments(int movieId, int page)
+        {
+
+            using var db = GetQuantityDbUserConnection();
+            return (await db.MovieComments.Where(comment => comment.MovieId == movieId)
```
Comment (lukisko) [inline Infrastructure/Repositories/CommentRepository.cs:24]: Really like this elegant approach.

File: Core/Interfaces/ICommentService.cs [hunk 3]
```diff
@@ -0,0 +1,10 @@
+﻿using MovieFiles.Core.Models;
+
+namespace MovieFiles.Core.Interfaces
+{
+    public interface ICommentService
+    {
+        Task<List<Comment>> GetComments(int movieId, int page);
```
Comment (lukisko) [inline Core/Interfaces/ICommentService.cs:7]: Consider to return object that will include information about total number of pages

**Pull request merged**